torch/csrc/distributed/c10d/ProcessGroupGloo.cpp

```diff
@@ -198,7 +198,7 @@ ReduceFunc toFunction(const ReduceOp& r) {
     case ReduceOp::PREMUL_SUM:
       TORCH_CHECK(false, "Cannot use ReduceOp.PREMUL_SUM with Gloo");
       break;
-    case ReduceOp::UNUSED:
```

Comment:
I don't like this change as right now we can get compile warnings if we forget to handle a case, this will not be the case with this, right?

Reply:
Fixed